### ISSUE TITLE
en21x setting

### ISSUE THREAD

**Issue by vitusbenson** [earthnet2021/earthnet-models-pytorch#15]
Add new setting en21x:

- [x] dataloader
- [x] metric
- [x] refactor log_viz
- [x] pixelwise
- [x] non-masked loss
